Jenkinsfile

```diff
@@ -280,6 +280,16 @@ def runBuild() {
     graknNode { workspace ->
         checkout scm
 
+        // Push to Grakn Maven if tests pass
+        if (!isMainBranch()) {
```

Comment:
This means we only deploy if it's a pull request. Is that right?

Reply:
copy/pasta fail